programs/_updown.xfrm/_updown.xfrm.in

```diff
@@ -425,13 +443,20 @@ nameserver ${nameserver}"
 	done
 
 	echo "${new_resolv_conf}" > "${ETC_RESOLV_CONF}"
-	rc=$?
```

Comment:
when resolve.conf is read-only this would fail, hence `rc=$?` to capture the result